src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectReportsController.kt

```diff
@@ -142,6 +169,17 @@ class ProjectReportsController(
               )
         }
 
+    val systemMetricUpdates =
+        payload.systemMetrics.associate {
+          it.metric to
+              ReportMetricEntryModel(
+                  target = it.target,
+                  value = it.overrideValue,
+                  notes = it.notes,
+                  internalComment = it.internalComment,
+              )
+        }
```

Comment:
Probably not super important, but these _could_ be converted to a separate method since both are identical between the 2 endpoints